woocommerce/woocommerce-admin
# Onboarding: Add purchase products task list item

Fixes #3442

* Adds a task list item for purchasing after the profiler.
* Adds the cart modal to the task list page.
* Removes already installed items from the cart.

### Screenshots
<img width="924" alt="Screen Shot 2019-12-24 at 8 23 24 PM" src="https://user-images.githubusercontent.com/10561050/71413288-6cbfaf80-268c-11ea-8051-e96f5e204a8d.png">
<img width="1129" alt="Screen Shot 2019-12-24 at 7 55 17 PM" src="https://user-images.githubusercontent.com/10561050/71413289-6d584600-268c-11ea-898b-136842058e1c.png">


### Detailed test instructions:

1. Install at least one plugin from the product types.
2. Run through the profiler, selecting product types and/or a theme that requires purchase.
3. Make sure the cart modal at the end of the theme selection continues to work as expected.
4. Make sure the cart includes items that require purchase, but not already installed plugins.
5. Continue to the task list without purchasing.
6. Note a task list item exists for purchasing items.
7.  Finish purchase and/or install the necessary plugins required.
8. Make sure the task list item is shown as completed and clicking the completed task list item does not open the cart modal.

Question: Do we want to show this task list item as complete or just hide it after complete (as it is now).
